lib/app-components/addon/components/branded-navbar/template.hbs

```diff
@@ -35,7 +35,7 @@
                         <li>
                             <OsfLink
```

Comment:
Hey @futa-ikeda this likely should have aria-labels. Also, are the list elements focusable?